projects/venofinance/index.js

```diff
@@ -30,24 +30,24 @@ async function tvlEra(api) {
   return api.getBalances()
 }
 
-async function tvlEthereum(api) {
-  const cro_pooled = await api.call({ abi: "uint256:totalPooledCro", target: zkCRO_contract_address });
-  api.addCGToken('crypto-com-chain', cro_pooled / 1e8);
-  const eth_pooled = await api.call({ abi: "uint256:totalPooledEth", target: ybETH_contract_address });
-  api.addCGToken('ethereum', eth_pooled / 1e18);  
-  const usd_pooled = await api.call({ abi: "uint256:totalUsdValue", target: ybUSD_contract_address });
-  api.addCGToken('dai', usd_pooled / 1e18); 
 
-  return api.getBalances();
-}
+async function tvlCronosZkEVM(zkEvmApi) {
+  const totalVEth = await zkEvmApi.call({ abi: "uint256:ybEthValue", target: vETH_contract_address });
+  zkEvmApi.addCGToken('ethereum', totalVEth / 1e18); 
+  const totalVUsd = await zkEvmApi.call({ abi: "uint256:ybUsdValue", target: vUSD_contract_address });
+  zkEvmApi.addCGToken('dai', totalVUsd / 1e18);
 
-async function tvlCronosZkEVM(api) {
-  const totalVEth = await api.call({ abi: "uint256:ybEthValue", target: vETH_contract_address });
-  api.addCGToken('ethereum', totalVEth / 1e18); 
-  const totalVUsd = await api.call({ abi: "uint256:ybUsdValue", target: vUSD_contract_address });
-  api.addCGToken('dai', totalVUsd / 1e18);
+  const ethApi = new sdk.ChainApi({ chain: 'ethereum' });
+  const cro_pooled = await ethApi.call({ abi: "uint256:totalPooledCro", target: zkCRO_contract_address });
+  const cronosApi = new sdk.ChainApi({ chain: 'cronos' });
+  const totalLcro = await cronosApi.call({
+    abi: 'function convertToShare(uint256 assets) view returns (uint256 shares)',
+    target: lcro_contract_address,
+    params: [cro_pooled]
+  });
 
-  return api.getBalances();
+  zkEvmApi.addCGToken('liquid-cro', totalLcro / 1e8);
```

Comment:
deduping vETH and vUSD I sort of understand, but this part, no. The interaction happens in ethereum right? why is it counted in cronos zkevm?

Reply:
I understand your concern. Please see the updated code. We read the zkCRO supply directly on the Cronos zkEVM chain now.

Reply:
no, the issue remains, I didnt see any option for the users to interact with zk chain on the webpage